cordova-lib/spec-plugman/platforms/windows.spec.js

```diff
@@ -50,18 +50,30 @@ function copyArray(arr) {
 }
 
 beforeEach(function () {
-    this.addMatchers({
+    jasmine.addMatchers({
         toContainXmlPath: function (xpath) {
             var xml = this.actual;
             var notText = this.isNot ? 'not ' : '';
-            this.message = function () {
-                return 'Expected xml \'' + et.tostring(xml) + '\' ' + notText + 'to contain elements matching \'' + xpath + '\'.';
-            };
-
-            return xml.find(xpath) !== null;
-        }    });
+            // this.message = function () {
+            //     return 'Expected xml \'' + et.tostring(xml) + '\' ' + notText + 'to contain elements matching \'' + xpath + '\'.';
+            // };
+            return {
+                compare: function(actual, expected) {
+                    var result = {};
+                    result.pass = fs.existsSync(actual);
+
+            //return xml.find(xpath) !== null;
+                    if(result.pass) {
+                        result.message = "expected " + actual + " to not exist";
+                    } else {
+                        result.message = "expected " + actual + " to exist";
```

Comment:
update second message to 

`'Expected xml \'' + et.tostring(xml) + '\' ' +' to contain elements matching \'' + expected + '\'.';`